apps/ledger-live-desktop/tests/models/AccountPage.ts

```diff
@@ -43,4 +43,17 @@ export class AccountPage {
     await this.stakeBannerButton.scrollIntoViewIfNeeded();
     await this.stakeBannerButton.click();
   }
+
+  async scrollToOperations() {
+    await this.page.locator("id=operation-list").scrollIntoViewIfNeeded();
```

Comment:
could this be a `data-test-id`?

Reply:
I've just moved some code from gre to the right page object, I assumed it wasn't part of my scope for this PR.
We'll change that if needed in a maintenance task later 👍